site2/docs/security-token-admin.md

```diff
@@ -8,38 +8,37 @@ sidebar_label: Token authentication admin
 
 Pulsar supports authenticating clients using security tokens that are based on [JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
 
-You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that
-is permitted to do some actions (for example, publish to a topic or consume from a topic).
+You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that is permitted to do some actions (for example, publish to a topic or consume from a topic).
 
-A user typically gets a user a token string from the administrator (or some automated service).
+A user typically gives a user a token string from the administrator (or some automated service).
```

Comment:
A user typically gets a token string from the administrator (or some automated service).